src/v/datalake/fwd.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2024 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
```

Comment:
I am not sure why, but i see two license used in the datalake coordinator one Enterprise and this BSL, which one should we use ?

Reply:
Updated licenses